addons/loyalty/models/loyalty_reward.py

```diff
@@ -50,7 +50,9 @@ def _compute_display_name(self):
     company_id = fields.Many2one(related='program_id.company_id', store=True)
     currency_id = fields.Many2one(related='program_id.currency_id')
 
-    description = fields.Char(compute='_compute_description', readonly=False, store=True, translate=True)
+    description = fields.Char(
+        compute='_compute_description',
+        readonly=False, store=True, translate=True, required=True, precompute=True)
```

Comment:
```suggestion
    description = fields.Char(
        translate=True, 
        compute='_compute_description', 
        precompute=True,
        store=True,
        readonly=False, 
        required=True
    )
```
According to our guidelines =]

Reply:
Oops, my bad!
Done with the changes. :smile:

Reply:
![image](https://github.com/user-attachments/assets/c54ddbf7-b88c-46c4-9750-d58ae5325c74)